lib/fs/fs.mli

```diff
@@ -4,8 +4,15 @@ module Filec = Filec
 
 (** A definition of a file tree. *)
```

Comment:
One minor addition while we're on this: let's improve docs by adding a note saying something like that `contents` and `file_type` are stored separately as two lazy values so we can quickly understand the file type and use a proper icon without reading the whole contents.

Just a note for future selves so we don't forget the motivation 😅